blockchain/process_test.go

```diff
@@ -0,0 +1,46 @@
+package blockchain_test
```

Comment:
Same for this file. What was your reasoning for naming this `blockchain_test`?

Reply:
Same reasoning as it is for file common_test.go